.github/workflows/missing-types.yml

```diff
@@ -0,0 +1,26 @@
+name: Check missing types
+
+on:
+  pull_request:
```

Comment:
any reason why we left it empty?

Reply:
Because we don't need to make any changes in the configuration of `pull_request`. I guess it can also be written as
```yml
on:
  - pull_request
```
but I wasn't sure.

Reply:
Fixed this.